pages/common/scp.md

```diff
@@ -1,30 +1,28 @@
 # scp
 
-> Copies files between hosts on a network.
-> Works over a secure connection (SSH).
+> Secure copy.
+> Copy files between hosts using Secure Copy Protocol over SSH.
 
-- Upload a file, or upload and rename a file:
+- Copy a loocal file to a remote host:
 
-`scp {{/local/file.txt}} {{10.0.0.1}}:{{/remote/path/}}`
-`scp {{/local/file.txt}} {{10.0.0.1}}:{{/remote/path/newname.txt}}`
+`scp {{local_file}} {{remote_host}}:{{/path/remote_file}}`
 
-- Download a file:
+- Copy a file from a remote host to a local folder:
 
-`scp {{10.0.0.1}}:{{/remote/path/file.txt}} {{/local/folder}}`
+`scp {{remote_host}}:{{/path/remote_file}} {{/path/local_dir}}`
 
-- Upload or download a directory:
+- Recursively copy the contents of a directory on a remote host to a local directory:
 
-`scp -r {{/local/folder}} {{10.0.0.1}}:{{/remote/path/}}`
-`scp -r {{10.0.0.1}}:{{/remote/path}} {{/local/folder}}`
+`scp -r {{remote_host}}:{{/path/remote_dir}} {{/path/local_dir}}`
 
-- Specify username on host:
+- Copies a file between two remote hosts transferring through the local host:
```

Comment:
The default is equivalent to running `ssh host1 "scp /path/filename user@host2:/path/filename" and so requires host1 to allow outgoing SSH and host2 to accept connections from host1.

Which might be sane with passwords on a simple network, but with SSH keys it requires a Agent Forwarding (dangerous) to function.